src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
```

Comment:
The class `ColorSupplier` should not begin with an empty line. Please remove the empty line to adhere to Java best practices.